### PR TITLE
Issue-79: Pass in coordinate variables

### PR DESCRIPTION
**Pull request by skorper**
Github Issue: #79 

### Description

Added optional coordinate variable params to subset function

### Overview of work done

Added new coordinate variable params (lat,lon,time) that can optionally be provided by user. If provided, don't manually compute coordinate variables. If not provided, manually compute.

### Overview of verification done

New unit test + existing tests all passing

### Overview of integration done

N/A

## PR checklist:

* [X] Linted
* [X] Updated unit tests
* [X] Updated changelog
* [ ] Integration testing

_See [Pull Request Review Checklist](../CONTRIBUTING.md#reviewing) for pointers on reviewing this pull request_

### CODE REVIEW

File: tests/test_subset.py [hunk 1]
```diff
@@ -1723,3 +1723,63 @@ def test_empty_temporal_subset(self):
...
+            time_var_names=None
+        )
+
+        assert lats == actual_lats
+        assert lons == dummy_lons
+        assert times == actual_times
+
+        # When only time is passed in, lats and lons are computed manually
+
+        # When time, lats, and lons are passed in, nothing is computed manually
```
Comment (frankinspace) [inline tests/test_subset.py:1775]: were you intending to add this test case as well? I see the "When only time is passed in" case but not this one

Comment (skorper) [reply tests/test_subset.py:1775]: Whoops, thanks I missed that one

File: podaac/subsetter/subset.py [hunk 2]
```diff
@@ -1077,9 +1077,39 @@ def walk_h5py(data_new, group):
     return nc_dataset, has_groups
 
 
+def get_coord_vars(dataset, lat_var_names=None, lon_var_names=None, time_var_names=None):
```
Comment (frankinspace) [inline podaac/subsetter/subset.py:1080]: Just a minor code comprehension suggestion. We now have `get_coord_vars` and `get_coord_variable_names` functions which seem at first look to do the same thing. Is there a better name for these functions that more accurately reflects their purpose? Maybe `get_coordinate_variable_names` for this function and `find_coordinate_variable_names` or `compute_coordinate_variable_names` for the other?

Comment (skorper) [reply podaac/subsetter/subset.py]: Thanks @frankinspace , I agree this makes a lot more sense. I've renamed them as follows:

* `get_coord_vars` -> `get_coord_variable_names`
* `get_coord_variable_names` -> `compute_coordinate_variable_names`
* `get_time_variable_name` -> `compute_time_variable_name`

**Review by frankinspace — APPROVED** [review]

**Pull request merged**